### ISSUE TITLE
About labels

### ISSUE THREAD

**Issue by JasonBoy1** [cvlab-stonybrook/DewarpNet#10]
Dear author:
Thank you for your username and password ! Thank you very much !

In your paper, you said that your produce 6 groups of images (annotations): (a) images, (b) albedo maps, (c) UV maps, (d) 3D coordinate maps, (e) Surface normals, (f) depth maps.  But you also mentioned backward map and shading map, which are not in those 6.

In the download link, you have 8 groups of files.
![image](https://user-images.githubusercontent.com/33180378/77255542-53d29480-6ca3-11ea-8d23-a46a7065e2a5.png)

**I am curious about:** 

1.  what's backward map?  There is no code to render backward map.
2. What's shading map? There is no code to render shading map. You also did not produce the code of refinement network.
3. what's the relationship between UV map and backward map? 
4. What's Ic. How to render it? (The paper said Ic is the I in checkerboard pattern texture.)
![image](https://user-images.githubusercontent.com/33180378/77256025-9184ec80-6ca6-11ea-8f75-4ab09b9c8afa.png)

5. What's recon.zip? I have not found any clue of this file. Maybe it is not used in the trianing
?
6. In your code, the training is not end-to-end, and the inference is combining the Shape Network and Texture Mapping Network. In the training code of Texture Mapping Network, you directly used the GT of coordinate map as input, not the output of Shape Network. Why did you call this end-to-end training in your paper?

**Comment by JasonBoy1** [comment]
Sorry about so many questions. I am looking forward to get your reply. Thanks a lot.

**Comment by mahendrathapa** [comment]
@WenjiaWang0312 I am in the same phase as you were previously. Could you please share your finding?
